model/qti/parser/ElementIdsExtractor.php

```diff
@@ -45,8 +51,14 @@ public function extract(Item $qtiItem, string $elementClass, string $attributeNa
         $this->ids = [];
 
         foreach ($qtiItem->getComposingElements($elementClass) as $element) {
+            $rawId = $element->attr($attributeName);
+
+            if ($this->onlyMediaManager && !$this->isMediaManagerMedia($rawId)) {
```

Comment:
I would suggest to leave the resolver to deals with media serial. However you can use the resolver to identify is the source is a mediaSource or localSource:
```code
$mediaAsset = $this->getMediaResolver()->resolve($rawId);
$mediaSource = $mediaAsset->getMediaSource();
$isMedia = ($mediaSource instanceof MediaSource);
```

Reply:
I think I cannot do this, cause I would need to include taoMediaManager as dependency of taoQtiItem

Reply:
oh right

Reply:
imo `MediaService::SCHEME_NAME . '://mediamanager';` is an hidden dependency. wdyt about moving this behavior to media manager?

Reply:
Ok, the responsibility was moved to media manager :)